hera_cal/utils.py

```diff
@@ -0,0 +1,33 @@
+import os
+def get_HERA_aa(freqs,calfile='hera_cm',
+                locations_file=None,
+                array_epoch_jd=None):
+    # create an aipy AntennaArray object with the HERA configuration as it
+    # was on array_epoch_jd.
+    #
+    # A default file is supplied but will soon be out of date.
+    #   - create a new one with hera_mc/scripts/XXX.py
```

Comment:
This code has been updated and tested. Comments have been improved here substantially.